cmd/format/format.go

```diff
@@ -77,6 +77,13 @@ func Run(v *viper.Viper, statz *stats.Stats, cmd *cobra.Command, paths []string)
 		}()
 	}
 
+	// clear the cache if desired before attempting to open its\
+	if cfg.ClearCache && !cfg.NoCache { // clear the cache if desired
```

Comment:
nit: comments got a bit wonky here:

```suggestion
	// clear the cache if desired before attempting to open it
	if cfg.ClearCache && !cfg.NoCache {
```

Also, IMO it's weird to not clear the cache if `--no-cache` is specified. I'd suggest one of the following instead:

1. Just clear the cache, OR
2. Print out a warning explaining why we're not clearing the cache despite `--clear-cache` being specified.